www/content/attributes/hx-swap-oob.md

```diff
@@ -27,14 +27,50 @@ The value of the `hx-swap-oob` can be:
 
 If the value is `true` or `outerHTML` (which are equivalent) the element will be swapped inline.
 
-If a swap value is given, that swap strategy will be used.
+If a swap value is given, that swap strategy will be used and the encapsulating tag pair will be stripped for all strategies other than `outerHTML`.
 
 If a selector is given, all elements matched by that selector will be swapped.  If not, the element with an ID matching the new content will be swapped.
 
-### Troublesome Tables
+### Using alternate swap strategies
+
+As mentioned previously when using swap strategies other than `true` or `outerHTML` the encapsulating tags are stripped, as such you need to excapsulate the returned data with the correct tags for the context.
+
+For a table that uses `<tbody>`:
```

Comment:
Maybe to make it extra clear, precise here that the goal is to swap a `<tr>` tag in eventually? (just like you explicitly mention `<li>` and `<p>` below!)

Reply:
I have update the language lmk what you think!